wasm/wasm-introduction.md

```diff
@@ -0,0 +1,157 @@
+# WASM Part I: What is WebAssembly? Reasons to take it seriously
+This article is the beginning of a series of two more posts that describes what is [WebAssembly(WASM)](https://webassembly.org/), the importance of this technology, and a simple [ClojureScript](https://clojurescript.org/) example running [WASM](https://webassembly.org/). On completion, the reader should understand the basic
```

Comment:
"two more posts" is not clear. Does this mean 1 of 3. How about "...a series that describes Webassembly..."